src/main/java/springboot/Model/Rubric.java

```diff
@@ -23,7 +23,7 @@ public Rubric(){
 
     public Rubric(Item item) {
         this.item = item;
-        item.setRubric(this);
+//        item.setRubric(this);
```

Comment:
we have version tracking so best to just delete this
```suggestion

```

Reply:
Will do

Reply:
In future, you can just click commit suggestion so I can see that you made the suggested change and we are tracking it